great_expectations/data_context/data_context.py

```diff
@@ -160,14 +145,11 @@ def __init__(self, project_config, context_root_dir, data_asset_name_delimiter='
     def _init_stores(self, store_configs):
         """Initialize all Stores for this DataContext.
 
-        TODO: Currently, most Stores are hardcoded.
-        Eventually, they should be read in from yml configs.
-        This will require some work on test fixtures.
-
         In general, Stores should take over most of the reading and writing to disk that DataContext had previously done.
         However, some files remain the responsiblity of the DataContext:
             great_expectations.yml
             plugins
+            credentials
```

Comment:
Credentials seems like a good store candidate, actually (with a filesystemstore as default)...why not?

Reply:
I suppose we could do it that way.

It seemed odd, because there would only be a single key (for a single file.)

I suppose we could change that and have it make changes to keys within the file itself...

If we go that route, then `great_expectations.yml` itself could be a store.

Reply:
This is a useful distinction...I wasn't thinking of the store as storing just the file, but rather as storing actual credentials. I'll have to look more closely at how stores are imagined here.